Adjust Makefile gzip command to support EL7 where `-k` switch is not supported.

--- a/Makefile	2020-01-13 15:58:18.226483772 +0100
+++ b/Makefile	2020-01-13 16:22:12.706810008 +0100
@@ -59,5 +59,5 @@
 earlyoom.1.gz: earlyoom.1
 ifdef PANDOC
-	gzip -f -k $<
+	gzip -f -c $< > $@
 endif
 
